feature-libs/order/components/replenishment-order-details/replenishment-order-cancellation/replenishment-order-cancellation.component.ts

```diff
@@ -44,6 +44,7 @@ export class ReplenishmentOrderCancellationComponent implements OnDestroy {
       this.vcr
     );
 
+    /* istanbul ignore if */
```

Comment:
Do we really want to ignore code coverage here? it's not our code.

Reply:
I think yes, this conditions only subcribe the dialog instance for release and there is no point to cover it with the test, but it decreases the test coverage significantly